src/devices/Ft232H/samples/Program.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Device.Spi;
+using System.Threading;
+using System.Device.Gpio;
+using Iot.Device.Bmxx80;
+using Iot.Device.Bmxx80.FilteringMode;
+using Iot.Device.Common;
+using Iot.Device.Ft232H;
+using Iot.Device.FtCommon;
+using UnitsNet;
+
+Console.WriteLine("Hello FT232H");
+var devices = FtCommon.GetDevices();
```

Comment:
What confuses me is that old (FT4222) FtCommon became Ft4222Common and Ft232 is now FtCommon - should they be just a single FtCommon?